Recommendation/user_rating_data_db_dump.py

```diff
@@ -0,0 +1,32 @@
+
+
+import pandas as pd
+import pymongo
+
+## Connect to database:
+client = pymongo.MongoClient("#####")

```

Comment:
Can you put this string into an environment variable?

For that you need to install [python-dotenv](https://pypi.org/project/python-dotenv/). That way important variables don't get uploaded to GitHub. Then you would need to create a .env file with the MONGO_URL and put that file into .gitignore.

I've done that on the file similar-trails.py. Please check the usage.

```suggestion
from dotenv import load_dotenv
from pathlib import Path

load_dotenv(dotenv_path = Path(<PATH_TO_ENV_FILE>))
MONGO_URL = os.getenv('MONGO_URL')

client = pymongo.MongoClient(MONGO_URL)
```